lib/command.ts

```diff
@@ -231,60 +231,43 @@ export function command(
     if (command) {
       currentContext.commands.push(command);
       currentContext.fullCommands.push(commandHandler.original);
+    } else {
+      // A null command indicates we are running the default command,
+      // if this is the case, we should show the root usage instructions
+      // rather than the usage instructions for the nested default command:
+      yargs.getUsageInstance().freeze();
```

Comment:
I don't understand why this would have been switched from `.unfreeze()` to `.freeze()`.

`unfreeze()` makes sense to me, because it pops us up one level to before the command was run, `freeze()` doesn't make sense to me.